docs/conf.py

```diff
@@ -21,7 +21,17 @@
 import os
 import sys
 
-sys.path.insert(0, os.path.abspath('..'))
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+# The full version, including alpha/beta/rc tags.
+#  Note: If plasmapy.__version__ can not be defined then it is set to 'unknown'.
+#        However, release needs to be a semantic style version number, so set
+#        the 'unknown' case to ''.
+from plasmapy import __version__ as release
```

Comment:
It's spooky, but it looks like `black` moved this here.

Reply:
Yeah, this is definitely a PEP8 thing.  I had the same warnings on my IDE.  I'm torn about this though...yes having it at the top satisfies PEP8, but having it where it was is more readable IMHO.

It might be best to just move the full `release` and `version` declaration here.